mappings/net/minecraft/entity/LivingEntity.mapping

```diff
@@ -99,7 +106,7 @@ CLASS net/minecraft/class_1752 net/minecraft/entity/LivingEntity
 	METHOD method_7145 swingHand ()V
 	METHOD method_7146 tickHandSwing ()V
 	METHOD method_7147 getAttributeContainer ()Lnet/minecraft/class_1769;
-	METHOD method_7148 getDamageEnchantmentTarget ()Lnet/minecraft/class_1758;
+	METHOD method_7148 getGroup ()Lnet/minecraft/class_1758;
```

Comment:
sounds a bit vague

Reply:
actually it'll be better if you rename `DamageEnchantmentTarget` to `EntityGroup`. 
Its used by upstream

Reply:
> actually it'll be better if you rename `DamageEnchantmentTarget` to `EntityGroup`.
I already did that

Reply:
ah alright nvm then